react/src/components/molecules/EmergencyAlert/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import is from 'is';
+import './style.css';
+
+const EmergencyAlert = (props) => {
+  const { message, timeStamp, link } = props;
+  const theme = props.theme.length === 0 ? 'c-highlight' : props.theme;
```

Comment:
should we switch to using classnames here?

Reply:
I think we should, e.g.
```suggestion
    const alertClasses = classNames({
      ma__emergency-alert: true,
      [`ma__emergency-alert--${props.theme}`]: props.theme
  });
```